FreeRTOS-Plus/Source/FreeRTOS-Plus-CLI/FreeRTOS_CLI.c

```diff
@@ -282,41 +284,62 @@ const char *pcReturn = NULL;
 
 static BaseType_t prvHelpCommand( char *pcWriteBuffer, size_t xWriteBufferLen, const char *pcCommandString )

```

Comment:
Please describe how the behviour of this function differs from the original.

Reply:
"help command" displays detailed information about the team